common/src/main/java/org/apache/rocketmq/common/sysflag/MessageSysFlag.java

```diff
@@ -16,7 +16,21 @@
  */
 package org.apache.rocketmq.common.sysflag;
 
+import org.apache.rocketmq.common.compression.CompressionType;
+
 public class MessageSysFlag {
+
+    /**
+     * Meaning of each bit in the system flag
+     *
+     * | bit    | 7 | 6 | 5         | 4        | 3           | 2                | 1                | 0                |
```

Comment:
Start from 0 to 7 would be better to read

Reply:
> Start from 0 to 7 would be better to read

I think it would be better to be consistent with the actual physical order